llama-index-integrations/vector_stores/llama-index-vector-stores-tablestore/llama_index/vector_stores/tablestore/base.py

```diff
@@ -54,6 +54,7 @@ class TablestoreVectorStore(BasePydanticVectorStore):
 
     is_embedding_query: bool = True
     stores_text: bool = True
+    _vector_dimension: int = PrivateAttr(default=512)
```

Comment:
shouldn't the default be 1536? The default embedding dimension in the framework?

Reply:
Ah I see it was previously 512 😢 I would have prefered this never had a default tbh. But its ok